spec/features/user_sees_a_politicians_outings_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.feature "visitor visits '/politicians_name'" do
+  scenario "they can view all outings related to politician" do
+
+    politician = create(:politician)
+    outing_1, outing_2 = create_list(:outing, 2, politician: politician)
+
```

Comment:
@drod1000 @Dpalazzari @njgheorghita  At our next update, can we expand on the test case and bring back the removed `politician_2` and its `outings` and have confirmation that those outings are not listed on the page?